code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -29,8 +29,10 @@
 	maxHealth = 80
 	health = 80
 	damage_coeff = list(BRUTE = 1, BURN = 1.25, TOX = 1, CLONE = 1, STAMINA = 1, OXY = 1)
-	unsuitable_cold_damage = 8
-	unsuitable_heat_damage = 8
+	flammable = TRUE
+	status_flags = 0
```

Comment:
```suggestion
	status_flags = NONE
```

This also seems like an odd choice. Tarantulas can't be pushed, now that behavior is no longer unique to them, and even Xenomorphs can be pushed. Do we really want this? (Also, if you do plan on keeping it, you should remove it from tarantula now that they inherit from parent).

Reply:
It should be noted that when I say they can no longer be pushed, I mean they essentially have combat mode's protection of not being able to be walked into like they don't exist. Realistically, I don't think you should be able to run into a giant spider in order to move it.